src/rules/caisson-id-check.js

```diff
@@ -0,0 +1,248 @@
+/**
+ * @title Caisson ID Check
+ * @overview Validate US driver's licenses and international passports in real time.
+ * @gallery true
+ * @category marketplace
+ *
+ * [Caisson Integration guide](https://www.caisson.com/docs/integration/auth0/)
+ */
+
+async function caissonIDCheck(user, context, callback) {
+  const { Auth0RedirectRuleUtilities } = require("@auth0/rule-utilities@0.1.0");
+
+  //copy off the config obj so we can use our own private key for session token signing.
+  let caissonConf = JSON.parse(JSON.stringify(configuration));
+  caissonConf.SESSION_TOKEN_SECRET = configuration.CAISSON_PRIVATE_KEY;
+
+  const manager = {
+    creds: {
+      public_key: caissonConf.CAISSON_PUBLIC_KEY,
+      private_key: caissonConf.CAISSON_PRIVATE_KEY,
+    },
+    debug:
+      caissonConf.CAISSON_DEBUG &&
+      caissonConf.CAISSON_DEBUG.toLowerCase() === "true"
+        ? true
+        : false,
+    idCheckFlags: {
+      login_frequency_days: parseInt(caissonConf.CAISSON_LOGIN_FREQUENCY_DAYS),
```

Comment:
```suggestion
      login_frequency_days: parseInt(caissonConf.CAISSON_LOGIN_FREQUENCY_DAYS, 10),
```

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/parseInt